### PR TITLE
Update StakeWise info

### LINKED ISSUE #6460: Update StakeWise information
StakeWise is listed as one of the pool staking options at https://ethereum.org/en/staking/pools/

We would appreciate ticking the Trustless and Diverse Clients boxes for StakeWise. Here is why:

# Trustless

Non-custodial - StakeWise migrated to `0x01` withdrawal prefix in April 2021. All our validators are created with `0x0100000000000000000000002296e122c1a20fca3cac3371357bdad3be0df079` withdrawal credentials since then. The withdrawals will be received by the [Pool Escrow](https://etherscan.io/address/0x2296e122c1a20fca3cac3371357bdad3be0df079) contract and users will burn their sETH2, rETH2 tokens and withdraw ETH from it.

Rewards automatically updated - A set of decentralized oracles update the rewards on a daily basis. The oracle code is available at https://github.com/stakewise/oracle/blob/master/README.md

DAO committee controlling exit signatures - The StakeWise DAO committee controls the exit signatures for our node operators meaning the DAO can force exit validators if required (i.e. pool funds cannot be held hostage by node operators) https://docs.stakewise.io/node-operator/generate-deposit-data#4.-send-encrypted-key-shares

Smart contracts upgradeability - Smart contracts can only be upgraded following a successful DAO vote, underpinning the security/trustlessness of all the above points.

# Diverse Clients
https://www.rated.network/ gives an overview of the current client diversity and new node operators joining the network continue to use a diversified set of clients as detailed in the node operator onboarding posts:
- https://forum.stakewise.io/t/swip-10-onboard-cryptomanufaktur-as-node-operator-in-stakewise-with-250-keys-on-the-ethereum-network/684
- https://forum.stakewise.io/t/verihash-allocate-250-keys-on-the-ethereum-network/713
- https://forum.stakewise.io/t/stakewise-labs-allocate-250-keys-on-the-ethereum-network/719
- https://forum.stakewise.io/t/finoa-consensus-services-allocate-250-keys-on-the-ethereum-network/763

It is also worth flagging that as part of the deployment package for StakeWise node operators, operators are able to transition between clients in real-time should there ever be a need to change clients in an emergency. This is something that node operators are required to prove they can do in the final round of stress testing on our testnet before they are allowed onto the mainnet. The list of EL and CL clients they can migrate between can be found at https://github.com/stakewise/helm-charts/tree/main/charts

### PR DESCRIPTION
**Pull request by tsudmi**
Fixes #6460

### CODE REVIEW

File: src/data/staking-products.json [hunk 1]
```diff
@@ -546,9 +546,9 @@
       ],
       "isFoss": true,
       "hasBugBounty": false,
-      "isTrustless": false,
+      "isTrustless": true,
       "hasPermissionlessNodes": false,
-      "pctMajorityClient": null,
+      "pctMajorityClient": 43.1,
```
Comment (minimalsm) [inline src/data/staking-products.json:551]: Meta comment (unrelated to this PR) @wackerow can we keep this with even though we don't have a super majority? 

In the future would it need to evolve to cover > 1/3 clients?

**Review by minimalsm — APPROVED** [review]
Hey @tsudmi, thanks for the PR :-) LGTM.

@wackerow you mind having a look before I merge?

File: src/data/staking-products.json (hunk 1 above)
Comment (wackerow) [reply src/data/staking-products.json:551]: This is already a flag only for "majority" not "supermajority" implying the client would only need to have >50% of the network (still a risk).

We can think about a better way to rank these now that client diversity is improving. We may want to consider simply hiding this indicator if the network stats reach a certain quality threshold, where emphasis on client diversity isn't as critical. The goal here was to try and flag projects that were more strongly contributing to the supermajority problem, but this is already less critical.

Comment (wackerow) [reply src/data/staking-products.json:551]: Perhaps it's worth opening a discussion on this as well, similar to #6452 for the "Trustless" tag

**Review by wackerow — APPROVED** [review]

**Pull request merged**